src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = body.querySelector('.logo');
+
+function clickLogo(resolved) {
+  logo.addEventListener('click', () => {
+    const div = document.createElement('div');
+
+    div.classList.add('message');
+    div.textContent = resolved;
+    body.appendChild(div);
+  });
+}
```

Comment:
The `clickLogo` function adds an event listener to the `logo` element every time it is called. If `clickLogo` is called multiple times, this could result in multiple event listeners being added, leading to unexpected behavior. Consider adding the event listener only once, possibly outside the function or by checking if it has already been added.